### ISSUE TITLE
st_point_ex.py read a file that does not necessarily exist

### ISSUE THREAD

**Issue by shengjh** [zilliztech/arctern#234]
```
def run_st_point(spark):
    points_df = spark.read.json("/tmp/points.json").cache()
    points_df.createOrReplaceTempView("points")
    register_funcs(spark)
    spark.sql("select ST_Point(x, y) from points").show()
```
This code read a file named /tmp/points.json. Howerver, it will run failed If the file does not exist.

**Comment by nezha2017** [comment]
It should run `point_gen.py` first, then submit this job.

**Comment by shengjh** [comment]
Got it.

**Comment by shengjh** [comment]
I think they can be combined to a same file.

**Comment by nezha2017** [comment]
> I think they can be combined to a same file.

OK, put them into one file,and create a PR

**Comment by shengjh** [comment]
fix #235
